OasisDEX/oasis-borrow
# 🧬/🧿 Ajna promo cards for Product Hub

# Ajna promo cards for Product Hub

What is this, a crossover episode?
![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/e18a63f2-9a86-4115-b051-357de0fbb57e)

  
## Changes 👷‍♀️

- Created handler that parses Product Hub API response and returns `AjnaLP` promo cards collection,
- modified `PromoCard` component so it can accept one of of the following: icon, image, or tokens list,
- modified `PromoCard` loading state so it matches better actual content that most of the promo cards has,
- modified `PromoCard` content so it accepts raw strings or translation config for server side generated responses,
- modified `TokenGroup` component to allow it to display token groups at forced icon size.
  
## How to test 🧪

- In `ProductHubView` change`useProductHubData` param to `AjnaLP` in `useProductHubData` hook,
- go to Product Hub testpage and check all possible variations in natural language form and promo cards that they returns.
